packages/cli/src/imports.js

```diff
@@ -8,7 +9,19 @@ async function _import(modulePath) {
   try {
     m = await import(path.resolve(localNodeModules, modulePath))
   } catch (e) {
-    m = await import(modulePath)
+    try {
+      m = await import(modulePath)
+    } catch (e) {
+      consola.fatal(
```

Comment:
This error maybe false positive? For example when importing itself cause a runtime error. We may use error.code

Reply:
You're right